src/rust/ide/lib/view/src/graph_editor/component/documentation_view.rs

```diff
@@ -0,0 +1,159 @@
+//! Documentation view visualization generating and presenting Enso Documentation under
+//! the documented node.
+
+use crate::prelude::*;
+
+use crate::graph_editor::component::visualization::*;
+use crate::graph_editor::component::visualization;
+
+use enso_frp as frp;
+use ensogl::display::DomSymbol;
+use ensogl::display::scene::Scene;
+use ensogl::display;
+use ensogl::system::web;
+use ensogl::system::web::StyleSetter;
+
```

Comment:
no section. Please use sections. Under imports there should be at least one section if there is a code.